core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/KaigiAppRobot.kt

```diff
@@ -38,6 +39,15 @@ class KaigiAppRobot @Inject constructor(
             .captureRoboImage()
     }
 
+    // FIXME I am doing this because capture() does not take a screenshot of the Achievement screen.
+    // If you know how to fix it, please fix it.
+    fun captureWhenAchievementScreen() {
```

Comment:
How about naming captureFirstRootScreen()?
It seems that this function wait for the timing